### ISSUE TITLE
Email tab incomplete

### ISSUE THREAD

**Issue by ssilvert** [keycloak/keycloak-admin-ui#1290]
The Email tab has missing attributes:

- Authentication
- Username
- Password

**Comment by jonkoops** [comment]
@ssilvert It looks like these are present as far as I can tell. If you click the 'Authentication' toggle these fields appear.

![image](https://user-images.githubusercontent.com/695720/135881879-a9549b83-51ca-4ec0-ba05-595b5b10bbdd.png)

**Comment by jenny-s51** [comment]
@ssilvert @jonkoops yes I also checked out the email tab and am seeing these fields as well! I have already implemented this code and it lives here: https://github.com/keycloak/keycloak-admin-ui/blob/master/src/realm-settings/EmailTab.tsx#L329-L375
